packages/dd-trace/src/config.js

```diff
@@ -60,6 +60,14 @@ class Config {
     }
     this.reportHostname = String(reportHostname) === 'true'
     this.scope = platform.env('DD_CONTEXT_PROPAGATION') === 'false' ? scopes.NOOP : scope
+    this.integrationsDisabled = coalesce(
+      options.integrationsDisabled,
+      (typeof platform.env('DD_INTEGRATIONS_DISABLED') === 'string'
```

Comment:
It would be cleaner to store this in a variable instead of calling `platform.env` twice. Environment variables are also guaranteed to be strings so the check is not necessary.

You can also use the shortcut syntax instead of a ternary operator when the second value is `undefined` like so:

```js
// equivalent to DD_INTEGRATIONS_DISABLED ? DD_INTEGRATIONS_DISABLED.split(',') : undefined
integrationsDisabled = DD_INTEGRATIONS_DISABLED && DD_INTEGRATIONS_DISABLED.split(',')
```

Reply:
updated per other notes in this review